code/modules/projectiles/projectile.dm

```diff
@@ -139,29 +164,37 @@
 	var/impact_light_range = 2
 	var/impact_light_color_override
 
-	//Homing
+	// Homing
+	/// If the projectile is homing. Warning - this changes projectile's processing logic, reverting it to segmented processing instead of new raymarching logic
 	var/homing = FALSE
+	/// Target the projectile is homing on
 	var/atom/homing_target
-	var/homing_turn_speed = 10 //Angle per tick.
-	var/homing_inaccuracy_min = 0 //in pixels for these. offsets are set once when setting target.
+	/// Angles per move segment, distance is based on SSprojectiles.pixels_per_decisecond
+	/// With pixels_per_decisecond set to 16 and homing_turn_speed, the projectile can turn up to 20 pixels per turf passed
```

Comment:
```suggestion
	/**
	 * Angles per move segment, distance is based on SSprojectiles.pixels_per_decisecond
	 * With pixels_per_decisecond set to 16 and homing_turn_speed, the projectile can turn up to 20 pixels per turf passed
	 */
```

Reply:
I don't think that we need to bloat already big chunks of comments with additional lines for this, autodocs works with multiple ///s

Reply:
Oh ok